### ISSUE TITLE
Would be nice if tables were sortable

### ISSUE THREAD

**Issue by tsdgeos** [AMDmi3/repology#174]
I would like to be able to go to https://repology.org/metapackage/okular/versions and click on the "Version" header and then get the versions sorted from newer to older

**Comment by AMDmi3** [comment]
Good idea. This would require our clever version comparison routine reimplemented in JS though, which I've had in far plans.

**Comment by anton-harvey** [comment]
Couldn't you assign an integer to each version using your back-end clever version comparison routine, pass it to the front-end, and sort on it using JavaScript? For example, the latest version would have `1`, the second latest version would have `2`, etc.
